pywr/optimisation/platypus.py

```diff
@@ -135,33 +135,62 @@ def evaluate(self, solution):
 class PywrRandomGenerator(platypus.RandomGenerator):
     """A Platypus Generator that injects the current setup of the Pywr model into the population.
```

Comment:
Need to update this line as you now use the class without injecting the current setup

Reply:
Done.